<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>user</class>
 <widget class="QMainWindow" name="user">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1495</width>
    <height>814</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>200</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgba(255, 255, 255, 150); /* Semi-transparent white */
    border: 2px solid black; /* Black border */
    border-radius: 8px;
    color: black; /* Default text color */
}

QPushButton:hover {
    background-color: rgba(255, 255, 255, 180); /* Slightly less transparent on hover */
    border: 2px solid black; /* Keep border black */
}
</string>
    </property>
    <property name="text">
     <string>supprimer</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="supprimer">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>200</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnequipement">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>370</y>
      <width>161</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>equipement</string>
    </property>
   </widget>
   <widget class="QPushButton" name="logout">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>640</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="logo">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>161</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnpatient">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>190</y>
      <width>161</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>patient</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="uidLineEdit">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>150</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="employeeComboBox">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>150</y>
      <width>76</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnformation">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>460</y>
      <width>161</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>formation</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>240</y>
      <width>981</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_3">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>150</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>100</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgba(255, 255, 255, 150); /* Semi-transparent white */
    border: 2px solid black; /* Black border */
    border-radius: 8px;
    color: black; /* Default text color */
}

QPushButton:hover {
    background-color: rgba(255, 255, 255, 180); /* Slightly less transparent on hover */
    border: 2px solid black; /* Keep border black */
}
</string>
    </property>
    <property name="text">
     <string>statistique</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="recherche">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>150</y>
      <width>271</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>221</width>
      <height>661</height>
     </rect>
    </property>
    <property name="title">
     <string>Ajouter </string>
    </property>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>113</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Id_employe&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;nom&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>110</y>
       <width>113</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;prenom&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>170</y>
       <width>113</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;NumT&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;genre&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>440</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>230</y>
       <width>113</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>330</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;adresse_employe&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>460</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>600</y>
       <width>93</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: rgba(255, 255, 255, 150); /* Semi-transparent white */
    border: 2px solid black; /* Black border */
    border-radius: 8px;
    color: black; /* Default text color */
}

QPushButton:hover {
    background-color: rgba(255, 255, 255, 180); /* Slightly less transparent on hover */
    border: 2px solid black; /* Keep border black */
}
</string>
     </property>
     <property name="text">
      <string>ajouter</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>530</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>400</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;specialite&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>290</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>HOMME</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>FEMME</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="lineEdit_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>360</y>
       <width>113</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>430</y>
       <width>113</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="modifier">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>600</y>
       <width>93</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: rgba(255, 255, 255, 150); /* Semi-transparent white */
    border: 2px solid black; /* Black border */
    border-radius: 8px;
    color: black; /* Default text color */
}

QPushButton:hover {
    background-color: rgba(255, 255, 255, 180); /* Slightly less transparent on hover */
    border: 2px solid black; /* Keep border black */
}
</string>
     </property>
     <property name="text">
      <string>modifier</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_7">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>500</y>
       <width>113</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>470</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;salaire &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="email">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>550</y>
       <width>113</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_24">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>530</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:696;&quot;&gt;email&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnvaccins">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>280</y>
      <width>161</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>vaccins</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnrecherche">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>550</y>
      <width>161</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>recherche</string>
    </property>
   </widget>
   <widget class="QComboBox" name="trier">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>200</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="combo_stat">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>100</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>100</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgba(255, 255, 255, 150); /* Semi-transparent white */
    border: 2px solid black; /* Black border */
    border-radius: 8px;
    color: black; /* Default text color */
}

QPushButton:hover {
    background-color: rgba(255, 255, 255, 180); /* Slightly less transparent on hover */
    border: 2px solid black; /* Keep border black */
}
</string>
    </property>
    <property name="text">
     <string>chatbot</string>
    </property>
   </widget>
   <widget class="QWidget" name="stat" native="true">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>530</y>
      <width>691</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addUidToEmployee">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>150</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgba(255, 255, 255, 150); /* Semi-transparent white */
    border: 2px solid black; /* Black border */
    border-radius: 8px;
    color: black; /* Default text color */
}

QPushButton:hover {
    background-color: rgba(255, 255, 255, 180); /* Slightly less transparent on hover */
    border: 2px solid black; /* Keep border black */
}
</string>
    </property>
    <property name="text">
     <string>add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Pdf">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>100</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgba(255, 255, 255, 150); /* Semi-transparent white */
    border: 2px solid black; /* Black border */
    border-radius: 8px;
    color: black; /* Default text color */
}

QPushButton:hover {
    background-color: rgba(255, 255, 255, 180); /* Slightly less transparent on hover */
    border: 2px solid black; /* Keep border black */
}
</string>
    </property>
    <property name="text">
     <string>pdf</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>0</y>
      <width>421</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:36pt; font-weight:700; font-style:italic; color:#ffffff;&quot;&gt;EMPLOYES&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnemploye">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>161</width>
      <height>81</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;employe&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>employe</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_tri">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>200</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Tri par :</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1495</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
